### PR TITLE
Revert "change start- and pausebuttons in div"

### PR DESCRIPTION
**Pull request by chmette**
Reverts chmette/reactive-tetris#13

### CODE REVIEW

File: src/components/styles/StyledStartButton.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import styled from "styled-components";
 
-// because of the keyboard functionality of space (accept ) and enter (submit) to a button, we used a div in these place to prevent these.
-
-export const StyledStartButton = styled.div`
+export const StyledStartButton = styled.button`
```
Comment (olhanotolga) [inline src/components/styles/StyledStartButton.js:3]: I'll close this PR, ok? we have the button set to .div on the main

**Review by olhanotolga — CHANGES_REQUESTED** [review]
I'll close this PR, ok? we have the button set to .div on the main

File: src/components/styles/StyledStartButton.js (hunk 1 above)
Comment (chmette) [reply src/components/styles/StyledStartButton.js:3]: ok

**Comment by olhanotolga** [comment]
Closed because changes have been applied elsewhere

**Pull request closed without merging**